src/App.css

```diff
@@ -1,17 +1,299 @@
+:root {
+  --logocolor: #0290ff;
+  --textcolor: #121212;
+  --secondcolor: #4386bf;
+  --progresscolor: #e8e8e8;
+}
+
 body {
```

Comment:
- [ ] Kindly use the same fonts as on the original design,
  - also consider adding the divider between the `stat` and `chapter` and try to use the same sizes and margins. 
  - you can give the `add new book` form the same width as the books section
  - kindly add the `hr` between the` book section` and `add new book` form, this will make your design closer to the Zeplin design

|Your design|Zeplin design|
|-|-|
|![Screenshot_20220624_202346](https://user-images.githubusercontent.com/25227022/175655553-669a69b7-599f-4520-babd-2119e3b7712f.png)|![Screenshot_20220624_203429](https://user-images.githubusercontent.com/25227022/175655558-da181519-d444-4db9-981b-dd4dd40d6e8d.png)|